src/app/fsat/fsat.component.ts

```diff
@@ -154,11 +157,6 @@ export class FsatComponent implements OnInit {
         this.modificationIndex = undefined;
       }
     });
-
-    this.modalOpenSubscription = this.fsatService.modalOpen.subscribe(isOpen => {
```

Comment:
We need to keep this sub. It's published to by other modals in child forms (see fan-field-data.ts).